src/components/RelocationBox/RelocationBox.jsx

```diff
@@ -0,0 +1,391 @@
+import { React, useState } from 'react';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionPanel,
+  AccordionButton,
+  AccordionIcon,
+  ChakraProvider,
+  FormControl,
+  FormLabel,
+  Input,
+  Select,
+  Textarea,
+} from '@chakra-ui/react';
+
+import { BsFillArrowRightCircleFill, BsFillCheckCircleFill, BsXCircleFill } from 'react-icons/bs';
+
+import PropTypes from 'prop-types';
+import RelocateBoxIcon from '../BoxIcons/RelocateBoxIcon.svg';
+import SaveChangesIcon from '../BoxIcons/SaveChangesIcon.svg';
+import { FYABackend } from '../../common/utils';
+import RequestChangesPopup from '../AlertPopups/RequestChangesPopup/RequestChangesPopup';
+import RejectBoxPopup from '../AlertPopups/RejectBoxPopup/RejectBoxPopup';
+import ImageVector from '../BoxIcons/ImageVector.svg';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../BoxIcons/ImageVector.svg'.